packages/react/src/generators/init/init.spec.ts

```diff
@@ -1,23 +1,32 @@
 import { readJson, Tree } from '@nrwl/devkit';
-import { createTreeWithEmptyWorkspace } from '@nrwl/devkit/testing';
+import { createTestProject } from '../utils/testing';
 
 import { reactInitGenerator } from './init';
 
 describe('init', () => {
   let tree: Tree;
 
   beforeEach(() => {
-    tree = createTreeWithEmptyWorkspace();
+    tree = createTestProject();
   });
 
   it('should not add jest config if unitTestRunner is none', async () => {
     await reactInitGenerator(tree, { unitTestRunner: 'none' });
     expect(tree.exists('jest.config.ts')).toEqual(false);
   });
 
-  it('should not add jest config if unitTestRunner is none', async () => {
-    await reactInitGenerator(tree, { unitTestRunner: 'none' });
-    expect(tree.exists('jest.config.ts')).toEqual(false);
+  it('should add jest config if unitTestRunner is jest', async () => {
```

Comment:
The setup is fine so far, but I wouldn't say this test is needed. With this equal, you test the test setup and not the plugin